challenges/systems/challenge-2.md

```diff
@@ -1,14 +1,15 @@
 # Summary
 
-Implement a prototype load balancer.
+Implement a basic load balancer. Load balancer are used to distribute network
+load across hosts.
```

Comment:
Since we use the term upstream a lot in the rest of this document, how about:

"Load balancers are used to distribute network load across multiple hosts (or upstreams)."

Reply:
I commented about the same thing. Yeah, I think it needs clarification. Hosts may work or "upstream host".

Reply:
Done.